docs/packaging-applications/build-servers/jenkins/pipeline.md

```diff
@@ -40,17 +54,22 @@ _**octopusPushPackage** allows you to push packages to the package repository in
 
 | Parameters      | Required | Description |
 |-----------------|----------|-------------|
-| `toolId` | Yes | The Octopus CLI tool to use. |
-| `serverId` | Yes | The ID of the target server to push the package. |
-| `spaceId` | Yes | The ID Space on the server to push the package. |
+| `toolId` | Yes | The configured Octopus CLI tool to use. |
+| `serverId` | Yes | The configured Server Id of the target server to push the package. |
```

Comment:
I'm changing a few Id to ID, when quoting a parameter `severId` copying to case of the parameter is correct, but when talking about IDs we should use ID.
```suggestion
| `serverId` | Yes | The configured Server ID of the target server to push the package. |
```